README.md

```diff
@@ -17,24 +23,35 @@ your project.
-| [CLI](./cli)                                 | @aidanm3341, @lbulanti-ms, @willosborne |
-| [Spectral](./spectral)                       | @willosborne, @lbulanti-ms              |
-| [Visualizer](./visualizer)                   | @aidanm3341, @Budlee, @willosborne      |
+| Project                                      | Lead Maintainers                        | Builds                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|----------------------------------------------|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Common Architecture Language Model](./calm) | @rocketstack-matt, @jpgough-ms          | [![Validation of CALM Samples](https://github.com/finos-labs/architecture-as-code/actions/workflows/spectral-validation.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/spectral-validation.yml)                                                                                                                                                                                                             |
+| [Docs](./docs)                               | @rocketstack-matt                       | [![Build Docs](https://github.com/finos-labs/architecture-as-code/actions/workflows/docs-deploy.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/docs-deploy.yml) [![pages-build-deployment](https://github.com/finos-labs/architecture-as-code/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/pages/pages-build-deployment) |
+| [CLI](./cli)                                 | @aidanm3341, @lbulanti-ms, @willosborne | [![Build CLI](https://github.com/finos-labs/architecture-as-code/actions/workflows/cli-tests.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/cli-tests.yml)                                                                                                                                                                                                                                                  |
```

Comment:
I'm not worried about being here, but I think I should be added as a codeowner of CLI and Spectral folders. Do you want to add me in this PR?

Reply:
OOh . . . maybe we should have a vote . . . all those in favour of making @grahampacker-ms a codeowner thumbs up! 😄 

Will do it in a separate PR